PocketKit/Sources/PocketKit/Article/ReadableViewController.swift

```diff
@@ -93,6 +93,12 @@ class ReadableViewController: UIViewController {
         readableViewModel.fetchDetailsIfNeeded()
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
```

Comment:
Very subtly you can see that images may use more space then they should after a rotation, this attempts to correct that by making the collection view re-layout after a size change.